easy_my_coop_website_portal/controllers/main.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015-2016 Odoo S.A.
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
remove other contributors who didn't contributed

Reply:
I have put these Copyright because the function `_get_archive_groups()` is copied from their code (website_portal_v10 and website_portal).